src/main/java/com/linagora/james/mailets/GuessClassificationMailet.java

```diff
@@ -97,6 +111,12 @@ public GuessClassificationMailet() {
     @Override
     public void init() throws MessagingException {
         LOGGER.debug("init GuessClassificationMailet");
+        executorService = Executors.newFixedThreadPool(
+            MailetUtil.getInitParameterAsStrictlyPositiveInteger(
+                getInitParameter(THREAD_COUNT),
+                THREAD_COUNT_DEFAULT_VALUE));
```

Comment:
I think it is. If classification API is slow, then it can limit the throughtput of the mailet pipeline when the Executor is full.